screeninfo/_utils.py

```diff
@@ -11,8 +11,7 @@ def load_library(name):
     """Load a ctypes library and provide error handling"""
     path = ctypes.util.find_library(name)
     if not path:
-        raise FileNotFoundError('Could not locate library ' + name)
+        return None
```

Comment:
No, this will lead to meaningless errors such as `None type has no member drmModeGetResources` in other implementations.

This should throw an exception. I'd deal with this like this:

```
try:
    FileNotFoundError
except NameError:
    FileNotFoundError = RuntimeError
```

or, since this exception is then caught in some of the enumerators, create own exception type such as `LibraryNotFoundError`.

Reply:
In that case, I would put a Python 3 check function in utils and prevent `NameError` handling, as I do not think it is suitable for alternate code paths. Or, it would be possible to simply use a `RuntimeError` instead of a `FileNotFoundError`, to prevent Python version dependence altogether. Also, as far as I know a `FileNotFoundError` is not a child class of `RuntimeError` and thus two different kinds of exceptions can be thrown depending on version. Thus, I would suggest either of the following:
- Throwing an `OSError` independent of Python version (as that is also the error thrown in Python 2 when a file is not found).
- Using a `is_python_3()` function in `._utils` in order to prevent `NameError` catching and implement your solution.

Reply:
`OSError` sounds good enough :+1: